$OpenBSD$
Fix build. Upstream really wants to rule CMAKE_INSTALL_PREFIX. :(
--- CMakeLists.txt.orig	Tue Aug  5 12:20:19 2014
+++ CMakeLists.txt	Tue Aug  5 12:24:17 2014
@@ -1,18 +1,14 @@
 project(yagf C CXX)
 cmake_minimum_required(VERSION 2.6.0)
 #set(QT_QMAKE_EXECUTABLE /usr/bin/qmake-qt4)
-if (NOT DEFINED  ${CMAKE_INSTALL_PREFIX})
-set(CPACK_INSTALL_PREFIX /usr/)
-set (CMAKE_INSTALL_PREFIX /usr/)
-endif()
 set (LIB_PATH_SUFFIX )
 if (CMAKE_SIZEOF_VOID_P EQUAL 8)
-    if (EXISTS "${CMAKE_INSTALL_PREFIX}lib64")
+    if (EXISTS "${CMAKE_INSTALL_PREFIX}/lib64")
         set(LIB_PATH_SUFFIX 64)
-    endif (EXISTS "${CMAKE_INSTALL_PREFIX}lib64")
+    endif (EXISTS "${CMAKE_INSTALL_PREFIX}/lib64")
     set_property (GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE)
 endif(CMAKE_SIZEOF_VOID_P EQUAL 8)
-add_definitions( -DPRILIBRARY_PATH="${CMAKE_INSTALL_PREFIX}lib${LIB_PATH_SUFFIX}/yagf/")
+add_definitions( -DPRILIBRARY_PATH="${CMAKE_INSTALL_PREFIX}/lib${LIB_PATH_SUFFIX}/yagf/")
 
 # following 2 lines define where interface translations will be installed
 set(QML_DESTINATION share/yagf/translations/)
